### PR TITLE
Add fluid StethQueueConnector

### PR DESCRIPTION
**Pull request by shriyatyagii**
- Add fluid StethQueueConnector

### CODE REVIEW

File: contracts/mainnet/connectors/steth-queue/main.sol [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+        IFluidStETHQueue fluidStethContract_ = IFluidStETHQueue(
+            FLUID_STETH_QUEUE
+        );
+
+        stETHAmt_ = stETHAmt_ == type(uint256).max
+            ? TokenInterface(STETH).balanceOf(address(this))
+            : stETHAmt_;
+
+        approve(TokenInterface(address(FLUID_STETH_QUEUE)), STETH, stETHAmt_);
```
Comment (KaymasJain) [inline contracts/mainnet/connectors/steth-queue/main.sol:34]:
```suggestion
        approve(TokenInterface(address(STETH)), FLUID_STETH_QUEUE, stETHAmt_);
```

File: contracts/mainnet/connectors/steth-queue/main.sol [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        uint256 requestIdFrom_ = fluidStethContract_.queue(
+            ethBorrowAmount_,
+            stETHAmount_,
+            address(this),
+            address(this)
+        ); // todo: confirm
```
Comment (KaymasJain) [inline contracts/mainnet/connectors/steth-queue/main.sol:40]: This is fine!

Comment (shriyatyagii) [reply contracts/mainnet/connectors/steth-queue/main.sol:40]: Okay! great

File: contracts/mainnet/connectors/steth-queue/main.sol [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        );
+
+        stETHAmt_ = stETHAmt_ == type(uint256).max
+            ? TokenInterface(STETH).balanceOf(address(this))
+            : stETHAmt_;
+
+        approve(TokenInterface(address(STETH)), FLUID_STETH_QUEUE, stETHAmt_);
+        uint256 requestIdFrom_ = fluidStethContract_.queue(
+            ethBorrowAmount_,
+            stETHAmount_,
```
Comment (KaymasJain) [inline contracts/mainnet/connectors/steth-queue/main.sol:37]:
```suggestion
            stETHAmt_,
```

File: contracts/mainnet/connectors/steth-queue/main.sol [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            address(this),
+            address(this)
+        ); // todo: confirm
+
+        setUint(setId_, requestIdFrom_);
+        _eventName = "LogQueueSteth(uint256,uint256,uint256,uint256,uint256)";
+        _eventParam = abi.encode(
+            stETHAmount_,
```
Comment (KaymasJain) [inline contracts/mainnet/connectors/steth-queue/main.sol:45]:
```suggestion
            stETHAmt_,
```

File: contracts/mainnet/connectors/steth-queue/main.sol (hunk 3 above)
Comment (shriyatyagii) [reply contracts/mainnet/connectors/steth-queue/main.sol]: Resolved

File: contracts/mainnet/connectors/steth-queue/main.sol (hunk 4 above)
Comment (shriyatyagii) [reply contracts/mainnet/connectors/steth-queue/main.sol]: Resolved

**Comment by shriyatyagii** [comment]
Add connector ->

- `FLUID-STETH-QUEUE`:

**Pull request merged**